CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs

```diff
@@ -241,16 +241,20 @@ private void CloseImeComposition()
         private void OnImeComposition(IntPtr hwnd, int lParam)
         {
             string text = string.Empty;
-
+            var underlines = new List<CompositionUnderline>();
+            int compositionStart = 0;
```

Comment:
Instead of hosting these values up, just pass in an empty array and 0 to ImeSetComposition please. Passing in values directly makes the code clearer.

Move `underlines` and `compositionStart` back to their original place.